cypress/support/pages/home.pageObject.js

```diff
@@ -3,8 +3,23 @@ import PageObject from '../PageObject';
 class homePageObject extends PageObject {
   url = '/#/';
 
-  get usernameLink() {
-    return cy.getByDataCy('/profile/');
+  usernameLink(username) {
+    cy.getByDataCy(`/profile/${username}`)
+      .should('contain', username);
+  }
+
+  newUsernameLink(username) {
+    cy.getByDataCy(`/profile/${username}`)
+      .should('contain', username);
+  }
+
+  clickSettingsLink() {
```

Comment:
The `clickSettingsLink` method is missing a return statement. Add a return statement to return the result of the Cypress command.